.rubocop.yml

```diff
@@ -215,6 +215,46 @@ Lint/UselessAssignment:
 Lint/DeprecatedClassMethods:
   Enabled: true
 
+Lint/DeprecatedOpenSSLConstant:
+  Enabled: true
+
+Lint/DuplicateElsifCondition:
+  Enabled: true
+
+Lint/SelfAssignment:
+  Enabled: true
+  Exclude:
+    - actionpack/lib/action_dispatch/journey/gtg/transition_table.rb
```

Comment:
Excluding specific `lib` files feels one step removed from pragma comments, so I'm not sure if we want this.  (Excluding tests seems more justifiable to me though, depending on the rule.)